queue_job/models/base.py

```diff
@@ -8,7 +8,11 @@
 
 
 class Base(models.AbstractModel):
-    """ The base model, which is implicitly inherited by all models. """
+    """ The base model, which is implicitly inherited by all models.
```

Comment:
Shouldn't you remove the initial space the same as you have made with the other docstrings?

Reply:
sure, thanks for noticing